modules/acid/src/main/ts/ephox/acid/api/Main.ts

```diff
@@ -3,9 +3,4 @@ import * as HsvColour from './colour/HsvColour';
 import * as RgbaColour from './colour/RgbaColour';
 import * as ColourPicker from '../gui/ColourPicker';
 
-export {
-  HexColour,
-  HsvColour,
-  RgbaColour,
-  ColourPicker
-};
+export { HexColour, HsvColour, RgbaColour, ColourPicker };
```

Comment:
This is the opposite of what we usually try to do :/

Reply:
It does both depending on how many items there are which will be annoying for code reviews when you add one item and it switches styling.